components/NowPlaying.tsx

```diff
@@ -11,12 +12,13 @@ export const useNowPlaying = () => {
         '/api/now-playing',
         fetcher
     );
+    const { t } = useTranslation('home');
 
-    return data;
+    return { data, t };
 };
 
 const NowPlaying = () => {
-    const data = useNowPlaying();
+    const { data, t } = useNowPlaying();
```

Comment:
Why not just put the `useTranslation` call down here? The string isn't dependent on whether there is music playing, as the entire component will be hidden if nothing is.